src/components/ADempiere/Form/VPOS/Collection/index.vue

```diff
@@ -699,14 +726,14 @@ export default {
         containerUuid,
         columnName: 'ReferenceNo'
       })
-      this.amontSend = this.dayRate.divideRate * this.amontSend
       if (this.sendToServer) {
         this.$store.dispatch('setPaymentBox', {
           posUuid,
           orderUuid,
           bankUuid,
           referenceNo,
-          amount: this.amontSend * this.convertion,
+          amount: this.amontSend,
+          convertedAmount: this.amontSend * this.dayRate.divideRate * this.convertion,
```

Comment:
I don't know what is the reason of `this.amontSend * this.dayRate.divideRate * this.convertion` just you should have the converted from the collect calculation instead